index.ts

```diff
@@ -158,35 +212,29 @@ module.exports = class Filter extends Plugin {
   }
 
   async build() {
-    // @ts-ignore
     let srcDir = this.inputPaths[0];
-    // @ts-ignore
     let destDir = this.outputPath;
 
     if (this.dependencyInvalidation && !this.dependencies) {
-      this.dependencies = this.processor.initialDependencies(srcDir);
+      this.dependencies = this.processor.initialDependencies(srcDir, { fs: this.input });
```

Comment:
We still have dependency over the `this.inputPaths`. We will not be able to remove this dependency, is it?

Reply:
And dependency logic is more around considering `Filter` takes only one tree as input. #188  talks about an array of trees as an input. We probably have to revisit this part later to accommodate that change.

Reply:
It looks like addressed in #191. Thanks